mail_tracking/tests/test_mail_tracking.py

```diff
@@ -27,6 +27,11 @@ def __str__(self):
         return "Test suite"
 
 
+class FakeConnection:
+    def cursor(self):
+        return True
```

Comment:
Why this? I don't think this is necessary

Reply:
You're right, I've removed this, thanks!